docs/basics/integrations-overview.md

```diff
@@ -20,7 +20,7 @@ We partner with the top companies in the industry to bring you a complete soluti
 <div className="box-wrapper" markdown="1">
```

Comment:
Can we remove the 'box card' div class around the first paragraph under the CI/CD Tools heading? The card looks like a link when you hover over it.

![Screen Shot 2022-11-25 at 10 56 53 AM](https://user-images.githubusercontent.com/15352964/204020730-b029001e-e5d1-49bb-ad32-25b63fb9b06e.png)


<img width="1096" alt="Screen Shot 2022-11-25 at 10 48 06 AM" src="https://user-images.githubusercontent.com/15352964/204020169-1d1d2acb-e45d-4fe7-bc10-8017c28f7b4a.png">